modules/core/src/core/animation-loop.js

```diff
@@ -59,6 +59,8 @@ export default class AnimationLoop {
     this.gl = gl;
     this.needsRedraw = null;
 
+    this._stopped = true;
```

Comment:
I'm not a big fan of double negatives. Can we change this to something like `started`?

Reply:
How about `_running`? To me, `started === true` implies the loop has been started at some point in the past, even if it might not currently be running.